packages/fluentui/react-bindings/src/compose/next/types.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+/**
+ * Generic name to any dictionary.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type GenericDictionary = Record<string, any>;
+
+/**
+ * Class dictionary.
+ */
+export type ClassDictionary = Record<string, string>;
+
+export interface ComponentProps extends GenericDictionary {
+  as?: React.ElementType;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ref?: React.Ref<any>;
+  className?: string;
+}
+
+// Shorthand types
+
+export type ShorthandRenderFunction<TProps> = (Component: React.ElementType<TProps>, props: TProps) => React.ReactNode;
+
+export type ShorthandProps<TProps extends ComponentProps = {}> =
+  | React.ReactChild
+  | React.ReactNodeArray
+  | React.ReactPortal
+  | boolean
+  | number
+  | null
+  | undefined
+  | (TProps &
+      ComponentProps & {
+        children?: TProps['children'] | ShorthandRenderFunction<TProps>;
+      });
```

Comment:
Can you please check what stuff from `/next` directory is really used an delete unnecessary things?

Reply:
I would say let's just do the stupid thing and copy stuff... deleting stuff is not hard and we can do that later :)

Reply:
So I looked in to `/next` folder, and turns out nothing from it is used. 
Thanks Ling. But I will remove it because it introduces @fluentui/utilities dependency.